x-pack/plugins/observability_solution/observability_shared/public/hooks/use_breadcrumbs.ts

```diff
@@ -58,11 +94,26 @@ export const useBreadcrumbs = (
   }>();
   const setTitle = docTitle.change;
   const appPath = getUrlForApp(app?.id ?? 'observability-overview') ?? '';
+  const chromeStyle = useObservable(getChromeStyle$());
 
-  const setBreadcrumbs = useMemo(
-    () => serverless?.setBreadcrumbs ?? chromeSetBreadcrumbs,
-    [serverless, chromeSetBreadcrumbs]
-  );
+  const setBreadcrumbs = useMemo(() => {
+    if (serverless?.setBreadcrumbs && !classicOnly)
```

Comment:
Nit: Can we simplify that a bit and have the logic setting the conditions if we don't have `serverless?.setBreadcrumbs` and then the 2 follow-up conditions 

Something like:
```
     const setBreadcrumbs = useMemo(() => {
    if (!serverless?.setBreadcrumbs) {
      return (breadcrumbs: ChromeBreadcrumb[]) =>
        chromeSetBreadcrumbs(
          breadcrumbs,
          !classicOnly
            ? {
                project: {
                  value: breadcrumbs,
                  absolute,
                },
              }
            : undefined
        );
    }
    if (!classicOnly)
      return (breadcrumbs: ChromeBreadcrumb[]) =>
        serverless?.setBreadcrumbs(breadcrumbs, { absolute });
    if (classicOnly) return;
  }, [serverless, classicOnly, absolute, chromeSetBreadcrumbs]);
```

Wdyt?

Reply:
Can do 👍